easybuild/easyconfigs/h/HPX/HPX-1.3.0-CrayGNU-19.06-APEX.eb

```diff
@@ -0,0 +1,49 @@
+# Mikael Simberg (CSCS), John Biddiscombe (CSCS), Victor Holanda (CSCS)
+easyblock = 'CMakeMake'
+
+name = 'HPX'
+version = '1.3.0'
+checksums = ['cd34da674064c4cc4a331402edbd65c5a1f8058fb46003314ca18fa08423c5ad']
+
+homepage = 'http://stellar-group.org/libraries/hpx'
+description = """HPX (High Performance ParalleX) is a general purpose C++
+runtime system for parallel and distributed applications of any scale. It
+strives to provide a unified programming model which transparently utilizes
+the available resources to achieve unprecedented levels of scalability.
+"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.06'}
+toolchainopts = {'pic': True}
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+    ('Boost', '1.70.0', '-python3'),
+    ('hwloc', '2.0.3', '', True),
+    ('jemalloc', '5.1.0'),
+    ('papi/5.7.0.1', EXTERNAL_MODULE),
+    ('OTF2', '2.2'),
```

Comment:
Change according to the `hwloc` above since `OTF2` is generic

Reply:
Updated. Thanks!